buildSrc/src/main/groovy/org/elasticsearch/gradle/test/DistroTestPlugin.java

```diff
@@ -370,7 +370,8 @@ private static void addDistro(NamedDomainObjectContainer<ElasticsearchDistributi
             if (type == Type.ARCHIVE) {
                 d.setPlatform(platform);
             }
-            d.setBundledJdk(bundledJdk);
+            // We don't test Docker images with a non-bundled JDK
+            d.setBundledJdk(type == Type.DOCKER || bundledJdk);
```

Comment:
This should be handled outside of addDistro, otherwise we are adding the same distribution twice? ie around line 325 where this method is called